js/data.json

```diff
@@ -1262,6 +1268,18 @@ osts = [
 		"link" : "https://my.mixtape.moe/swjtom.mp3",
 		"name" : "Boku no Hero Academia S2 OST-2 Jet Set Run",
 		"img" : "https://images6.alphacoders.com/820/820024.png"
+	},
+
+	{
+		"link" : "https://s1.vocaroo.com/media/download_temp/Vocaroo_s1b0f75sq5EN.mp3",
+		"name" : "Kimi no na wa OST - Yumetourou Radwimps",
+		"img" : "https://www.koi-nya.net/img/subidos_posts/2017/01/Kimi-no-na-wa-protagonistas.jpg"
+	},
+
+	{
+		"link" : "https://s1.vocaroo.com/media/download_temp/Vocaroo_s1oMIbR1BFgl.mp3",
+		"name" : "Kimi no na wa OST - Zen zen zense Radwimps",
+		"img" : "https://www.koi-nya.net/img/subidos_posts/2017/01/Kimi-no-na-wa-protagonistas.jpg"
```

Comment:
Please use some other image for this one. Redundant images aren't very appealing.